tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchTemplateSearchTests.java

```diff
@@ -0,0 +1,257 @@
+package org.alfresco.elasticsearch;
+
+import static org.alfresco.elasticsearch.SearchQueryService.req;
+import static org.alfresco.utility.data.RandomData.getRandomFile;
+import static org.alfresco.utility.data.RandomData.getRandomName;
+
+import java.util.Map;
+
+import org.alfresco.rest.model.RestTagModel;
+import org.alfresco.rest.search.RestRequestDefaultsModel;
+import org.alfresco.rest.search.SearchRequest;
+import org.alfresco.tas.AlfrescoStackInitializer;
+import org.alfresco.utility.data.DataContent;
+import org.alfresco.utility.data.DataUser;
+import org.alfresco.utility.model.ContentModel;
+import org.alfresco.utility.model.FileModel;
+import org.alfresco.utility.model.FileType;
+import org.alfresco.utility.model.FolderModel;
+import org.alfresco.utility.model.TestGroup;
+import org.alfresco.utility.model.UserModel;
+import org.alfresco.utility.network.ServerHealth;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+@ContextConfiguration(locations = "classpath:alfresco-elasticsearch-context.xml",
+    initializers = AlfrescoStackInitializer.class)
+public class ElasticsearchTemplateSearchTests extends AbstractTestNGSpringContextTests
+{
+    private static final String SEARCH_TERM = "sample";
+
+    @Autowired
+    ServerHealth serverHealth;
+
+    @Autowired
+    DataUser dataUser;
+
+    @Autowired
+    DataContent dataContent;
+
+    @Autowired
+    SearchQueryService searchQueryService;
```

Comment:
Can't these fields be `private`?

Reply:
They should be private, somehow I've missed it 🤔 Thanks!